src/callstack.cr

```diff
@@ -17,6 +17,19 @@ require "callstack/lib_unwind"
   require "debug/dwarf"
 {% end %}
 
+# `caller` returns the current execution stack (a backtrace) in the form
```

Comment:
There's a whole section on documenting code in the crystal language ref [here](https://crystal-lang.org/docs/conventions/documenting_code.html).